lib/filter.g

```diff
@@ -171,6 +171,60 @@ BIND_GLOBAL( "InstallTrueMethodNewFilter", function ( tofilt, from )
 end );
 
 
+#############################################################################
+##
+#F SuspendMethodReordering()
+#F ResumeMethodReordering()
+#F ResetMethodReordering()
+##
+##  <#GAPDoc Label="MethodReordering">
+##  <ManSection>
+##  <Func Name="SuspendMethodReordering" Arg=""/>
+##  <Func Name="ResumeMethodReordering" Arg=""/>
+##  <Func Name="ResetMethodReordering" Arg=""/>
+##
+##  <Description> These functions control whether the method reordering process
+##  described in <Ref Func="InstallTrueMethod"/> is invoked or not. Since this 
+##  process can be comparatively time-consuming, it is usually suspended when
+##  a lot of implications are due to be installed, for instance when loading
+##  the library, or a package. This is done by called <C>SuspendMethodReordering()</C>
+##  once the installations are done, <C>ResumeMethodReordering()</C> should be called. 
+##  These pairs of calls can be nested. When the outermost pair is complete, method
+##  reordering takes place and is enabled in <C>InstallTrueMethod</C> thereafter.
+##
+##  <C>ResetMethodReordering()</C> effectively exits all nested suspensions, resuming
+##  reordering immediately. This function is mainly provided for error recovery and 
+##  similar purposes.</Description>
```

Comment:
Indeed, what if there is an error duing package loading? That happens frequently for me if I try `LoadAllPackages()` (because I have tons of non-distributed packages and development versions, with all kinds of conflicts). For options, we have the `OnQuit` hook. Perhaps a call to `ResetMethodReordering` should be added to `OnQuit`, too? (Both in the GAP and in the HPC-GAP version).